style.css

```diff
@@ -233,7 +237,7 @@ img {
   .banner.container {
     column-gap: 15px;
   }
-  .read-more-btn {
+  .primary-btn {
```

Comment:
why duplicate `primary-btn` class ?

Reply:
At first I needed only single button but now I added some more buttons so to style all buttons I renamed the branch to primary-btn

Reply:
I checked the code again and the duplicate classes are separated by media queries